com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -461,6 +461,7 @@ private static Options getCommandLineOptions() {
         addOption(options, "brhtml", "build-report-html", true, "Filepath where to save a build report as HTML", false);
 
         addOption(options, null, "build-server", true, "The build server (when using native extensions)", true);
+        addOption(options, null, "build-server-header", true, "Additional build server header to set", true);
```

Comment:
(thoughts for the future)
I think we need to start thinking about how to deal with things like this.
We don't want 100's of options for each little use case, here in bob.
E:g. should each plugin get their options added like:
"--plugin_name.setting1=foo"

Reply:
Yes, the number of options is growing quite a lot.

Reply:
Perhaps we could associate options with each command and do something like a per command help?

```sh
# show help and options for the build command
java -jar bob.jar help build
```